openfisca_us/parameters/gov/states/mo/tax/income/deductions/federal_income_tax_deduction_caps.yaml

```diff
@@ -0,0 +1,19 @@
+description: Federal income tax deduction caps by filing status.
+SINGLE: 
+  2021-01-01: 10_000
+JOINT: 
+  2021-01-01: 10_000
+SEPARATE:  
+  2021-01-01: 5_000
+HEAD_OF_HOUSEHOLD: 
+  2021-01-01: 10_000
+WIDOW: 
+  2021-01-01: 10_000
+metadata:
```

Comment:
add breakdown, also to any other parameters with breakdowns

Reply:
don't follow what you mean here